### PR TITLE
feat: Document bots

### LINKED ISSUE #10094: 🐣  Bots part 2
- [x] Commands (currently marked as Beta) should be marked as Deprecated #10151
- [x] Allow bots to react #10138 
- [x] Installed bots should be listed in the admin settings similarly like commands are currently
  - [x] Additionally with a link to documentation how to write them
  - [x] And a link to the list of available bots (currently planned to be hosted in the docs of Talk for the time being)

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* Fix #10094 

image

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not required
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: docs/bot-management.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+|----------------------|--------|----------------------------------------------------------------------------------------------|
+| `id`                 | int    | Unique numeric identifier of the bot on this server                                          |
+| `name`               | string | Display name of the bot shown as author when it posts a message or reaction                  |
+| `description`        | string | A longer description of the bot helping moderators to decide if they want to enable this bot |
+| `url`                | string | URL endpoint that is triggered by this bot                                                   |
+| `url_hash`           | string | Hash of the URL prefixed with `bot-` serves as `actor_id`                                    |
+| `state`              | int    | One of the [Bot states](constants.md#bot-states)                                             |
+| `error_count`        | int    | Number of consecutive errors                                                                 |
+| `last_error_date`    | int    | UNIX timestamp of the last error                                                             |
```
Comment (Antreesy) [inline docs/bot-management.md:30]: in `seconds` or `milliseconds`?

File: docs/bot-management.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| `last_error_message` | string | One of the [Bot states](constants.md#bot-states)                                             |
```
Comment (Antreesy) [inline docs/bot-management.md:31]: Duplicated description

File: docs/bot-management.md (hunk 1 above)
Comment (nickvergessen) [reply docs/bot-management.md:30]: in PHP and DB we only use seconds

File: docs/bot-management.md (hunk 2 above)
Comment (nickvergessen) [reply docs/bot-management.md]: Fixed

**Pull request merged**